MIGRATION_GUIDE.TXT

```diff
@@ -1,6 +1,17 @@
 MIGRATION GUIDE FROM GDAL 3.10 to GDAL 3.11
 -------------------------------------------
 
+- The OGRLayer::GetExtent(OGREnvelope*, int bForce) and
+  OGRLayer::GetExtent(int iGeomField, OGREnvelope*, int bForce) methods are
+  no longer virtual methods that are implemented by drivers. They are now
+  user facing methods (with the change that the ``int bForce`` is now a
```

Comment:
I'm confused about what "user-facing" means here -- they were always user-facing (public), no?

Reply:
I've reworded / simplified this